src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -339,10 +339,7 @@ public static OnnxModel CreateFromBytes(byte[] modelBytes)
         public static OnnxModel CreateFromBytes(byte[] modelBytes, int? gpuDeviceId = null, bool fallbackToCpu = false,
             IDictionary<string, int[]> shapeDictionary = null)
         {
-            var tempModelDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
-            Directory.CreateDirectory(tempModelDir);
-
-            var tempModelFile = Path.Combine(tempModelDir, "model.onnx");
```

Comment:
Just curious, was it necessary to change this?

Reply:
It was a single file in a directory. It wasn't adding much to it and there was already code to delete the file, but not the directory. So I just stopped creating the directory.